sonaranalyzer-dotnet/rspec/vbnet/S2077_vb.net.html

```diff
@@ -63,14 +64,19 @@ <h2>Questionable Code Example</h2>
     context.Query(Of User)().FromSql(query) ' Questionable
 End Sub
 </pre>
+<h2>Exceptions</h2>
+<p>No issue will be raised in the following cases:</p>
+<ul>
+  <li> The SQL query is a hard coded string. </li>
```

Comment:
TODO update rspec before merging

Reply:
FYI I've added a comment to the ticket for NicoPM suggesting that the wording of one of the exceptions is changed.